Altis_Life.Altis/core/functions/fn_startLoadout.sqf

```diff
@@ -6,23 +6,44 @@
     Description:
     Loads a custom loadout on player when he got a new life
 */
+private _side = call {
+    if (playerSide isEqualTo civilian) exitWith {"civ"};
+    if (playerSide isEqualTo west) exitWith {"cop"};
+    if (playerSide isEqualTo independent) exitWith {"med"};
+};
+
 private _level = call {
-    if (playerSide isEqualTo civilian) exitWith {
+    if (_side isEqualTo "civ") exitWith {
         if (life_is_arrested) then {
             "arrested"
         } else {
-            floor(random (count(missionConfigFile >> "Loadouts" >> str(playerSide))-1))
+            "random"
         };
     };
-    if (playerSide isEqualTo west) exitWith {
+    if (_side isEqualTo "cop") exitWith {
         FETCH_CONST(life_coplevel)
     };
-    if (playerSide isEqualTo independent) exitWith {
+    if (_side isEqualTo "med") exitWith {
         FETCH_CONST(life_mediclevel)
     };
 };
 
-player setUnitLoadout (missionConfigFile >> "Loadouts" >> str(playerSide) >> format["lvl_%1",_level]);
+if (_side isEqualTo "civ" && _level isEqualTo "random") then {
+    private _civLoadout = getUnitLoadout (missionConfigFile >> "Loadouts" >> "civ_level_random");
+    private _arr = [

```

Comment:
Use getArray to fetch the list of random clothing from Config_Loadouts - just to keep everything in the same place to avoid people editing this file unnecessarily.